<launch>
    <arg name="use_respeaker" default="true" />
	<node pkg="mr_voice" type="SpeakerNode.py" name="speaker" output="screen">
        <param name="rate" value="80" />
        <param name="volume" value="1.0" />
        <param name="lang" value="en-US" />
    </node>
    <node pkg="mr_voice" type="SpeechToTextNode.py" name="voice" output="screen">
        <param name="topic_audio_path" value="/respeaker/audio_path" />
        <param name="topic_text" value="/voice/text" />
        <param name="lang" value="yue-Hant-HK" />
    </node>

    <group if="$(eval use_respeaker == true)">
        <node pkg="mr_voice" type="RespeakerNode.py" name="respeaker" output="screen">
            <param name="buffer_size" value="10" />
            <param name="audio_directory" value="/tmp/speech/" />
        </node>
    </group>
    <group if="$(eval use_respeaker == false)">
        <node pkg="mr_voice" type="MicNode.py" name="respeaker" output="screen">
            <param name="buffer_size" value="10" />
            <param name="audio_directory" value="/tmp/speech/" />
        </node>
    </group>
</launch>
